### PR TITLE
[SYCLomatic #1465] Add test to check the rewording of DPCT1026

### PR DESCRIPTION
**Pull request by yihwang-nv**

### CODE REVIEW

File: behavior_tests/src/bt-check-warning1026/do_test.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# ====------ do_test.py----------------------------------- *- Python -* ----===#
```
Comment (tangjj11) [inline behavior_tests/src/bt-check-warning1026/do_test.py]:
```suggestion
# ===------- do_test.py ---------------------------------- *- Python -* ----===#
```

File: behavior_tests/src/bt-check-warning1026/test.cu [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// ====------ test.cu ------------------------------------- *- CUDA -* ----===//
```
Comment (tangjj11) [inline behavior_tests/src/bt-check-warning1026/test.cu]:
```suggestion
// ===------- test.cu ------------------------------------- *- CUDA -* ----===//
```

File: behavior_tests/src/bt-check-warning1026/test.cu [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  size_t n_device_tmp = 0;
+  int host_in[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+  int host_out[10];
+  cudaMalloc((void **)&device_in, n * sizeof(int));
+  cudaMalloc((void **)&device_out, n * sizeof(int));
+  cudaMemcpy(device_in, (void *)host_in, sizeof(host_in), cudaMemcpyHostToDevice);
+  cub::DeviceReduce::Sum((void *)device_tmp, n_device_tmp, device_in, device_out, n);
+  cudaMalloc((void **)&device_tmp, n_device_tmp);
+  cub::DeviceReduce::Sum((void *)device_tmp, n_device_tmp, device_in, device_out, n);
+  cudaMemcpy((void *)host_out, (void *)device_out, sizeof(host_out), cudaMemcpyDeviceToHost);
```
Comment (tangjj11) [inline behavior_tests/src/bt-check-warning1026/test.cu]: should better format

**Review by tangjj11 — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**